modules/web/client-model/src/main/scala/observe/ui/model/ObsSummary.scala

```diff
@@ -82,8 +82,7 @@ object ObsSummary:
       observingMode      <- c.get[Option[ObservingMode]]("observingMode")
       visualizationTime  <- c.get[Option[Timestamp]]("visualizationTime")
       posAngleConstraint <- c.get[PosAngleConstraint]("posAngleConstraint")
-      programReference   <-
-        c.downField("program").downField("reference").get[Option[ProgramReference]]("label")
+      obsReference       <- c.downField("reference").get[Option[ObservationReference]]("label")
```

Comment:
These may fail if `reference` is null

Reply:
Doesn't it return `None`? Anyway, this is defensive. Ready observations are expected to always have a `reference`.

Reply:
I can confirm that it returns `None` instead of failing.